components/littlefs/LICENSE.md

```diff
@@ -0,0 +1,165 @@
+Apache License
```

Comment:
possible to add littlefs as a submodule vs. copying all of the files into the repository?

Reply:
Would love to, can you help figure out how?